packages/gatsby-plugin-manifest/README.md

```diff
@@ -143,25 +142,22 @@ module.exports = {
     {
       resolve: `gatsby-plugin-manifest`,
       options: {
+        name: `GatsbyJS`,
+        short_name: `Gatsby`,
+        display: `standalone`,
         manifests: [
           {
-            name: `GatsbyJS`,
-            short_name: `GatsbyJS`,
             start_url: `/de/`,
             regex: `^/de/.*`,
             language: `de`,
             background_color: `#f7f0eb`,
             theme_color: `#a2466c`,
-            display: `standalone`,
           },
           {
-            name: `GatsbyJS`,
-            short_name: `GatsbyJS`,
             start_url: `/`,
```

Comment:
My thought is there's no need to explicitly define the root or default manifest here. This can all be assumed from the root of options and only additional manifests need be defined here.

Reply:
Like so
```
{
      resolve: `gatsby-plugin-manifest`,
      options: {
        name: `GatsbyJS`,
        short_name: `Gatsby`,
        display: `standalone`,
        icon: `src/images/icon.png`,
        start_url: `/`,
        background_color: `#663399`,
        theme_color: `#fff`,
        manifests: [
          {
            start_url: `/de/`,
            language: `de`,
            background_color: `#f7f0eb`,
            theme_color: `#a2466c`,
          },
        ],
      },
    },
```
you mean? Maybe we then rename `manifests` to something like `localized_manifests` or `additional_manifests` (probably too vague)

Reply:
Exactally. I'm torn on naming. Let's make sure to address @lekoarts comments on the issue for now.